pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -47,6 +76,234 @@ def get_average_of_unique_labels(labels, values):
         return mean_values.flatten()
     return mean_values
 
+class Interstitials:
+    """
+    Class to analyse interstitial sites
+
+    This class internally does the following steps:
+
+        1. Initialize grid points (or Voronoi vertices) which are considered as
+            interstitial site candidates.
+        2. Eliminate points within a distance from the nearest neighboring atoms as
+            given by `min_distance`
+        3. Initialize neighbor environment using `get_neighbors`
+        4. Shift interstitial candidates to the nearest symmetric points with respect to the
+            neighboring atom sites/vertices.
+        5. Kick out points with large neighbor distance variances
```

Comment:
Should explain a bit more our intent on why large distance variances are bad, maybe

```suggestion
        5. Kick out points with large neighbor distance variances; this eliminates "irregular" shaped interstitials
```